projects/core/src/user/store/selectors/delivery-countries.selectors.ts

```diff
@@ -1,46 +1,41 @@
 import { MemoizedSelector, createSelector } from '@ngrx/store';
-import * as fromFeature from './../reducers/index';
 import * as fromReducer from './../reducers/delivery-countries.reducer';
-import { UserState, DeliveryCountriesState } from '../user-state';
+import {
+  UserState,
+  DeliveryCountriesState,
+  StateWithUser
+} from '../user-state';
 import { Country } from '../../../occ/occ-models/index';
+import { getUserState } from './feature.selector';
 
 export const getDeliveryCountriesState: MemoizedSelector<
-  any,
+  StateWithUser,
   DeliveryCountriesState
 > = createSelector(
-  fromFeature.getUserState,
+  getUserState,
   (state: UserState) => state.countries
 );
 
 export const getDeliveryCountriesEntites: MemoizedSelector<
-  any,
-  any
+  StateWithUser,
+  { [isocode: string]: any }
```

Comment:
This `any` here could possibly be replaced by `Country`, but please check it first

Reply:
the types in user store were improved in the latest commit